coredns/assets/configuration/spec.yaml

```diff
@@ -10,7 +10,16 @@ files:
     - template: instances/openmetrics
       overrides:
         openmetrics_endpoint.required: false
+        openmetrics_endpoint.enabled: true
+        openmetrics_endpoint.display_priority: 2
```

Comment:
Is `display_priority` necessary?

Reply:
Yes, I set the display priority higher in reference to the `prometheus_url` display priority so that it'll be displayed above `prometheus_url`. Otherwise, `prometheus_url` will be displayed first or at the very bottom if no priority is specified.